src/objects/zcl_abapgit_object_intf.clas.abap

```diff
@@ -414,25 +388,31 @@ CLASS zcl_abapgit_object_intf IMPLEMENTATION.
           ls_clskey TYPE seoclskey,
           ls_intf   TYPE ty_intf.
 
-
     IF iv_step = zif_abapgit_object=>gc_step_id-abap.
       ls_intf = read_xml( io_xml ).
 
       IF ls_intf-vseointerf-clsproxy = abap_true.
         " Proxy interfaces are managed via SPRX
         deserialize_proxy( iv_transport ).
-        RETURN.
 
       ELSE.
-        deserialize_intf( is_intf      = ls_intf
-                          iv_package   = iv_package
-                          iv_transport = iv_transport ).
+        mi_object_oriented_object_fct->create(
+          EXPORTING
+            iv_package    = iv_package
+          CHANGING
+            cg_properties = ls_intf-vseointerf ).
 
         ls_clskey-clsname = ms_item-obj_name.
         lt_source = zif_abapgit_object~mo_files->read_abap( ).
```

Comment:
Would we like to move reading the source close to `ls_intf = read_xml( io_xml ).`?

Advantage: Reading the content from the files would be in in one location (maybe even dedicated method)
Disadvantage: It is also read in the case of proxy interfaces. There it seems to be not necessary

Any opinion?

Reply:
well, I think for now we dont want more complexity in this PR, already contains too many changes that are not strictly necessary for the refactoring. Again, this code is a critical part in abapGit